cybersecurity/offensive/c2/apollo.yml

```diff
@@ -0,0 +1,746 @@
+description: Apollo is a Windows agent written in C# using the 4.0 .NET Framework designed to be used in SpecterOps training offerings. The agent is designed to be used with the Mythic C2 server.
+
+functions:
+  assembly_inject:
+    description: Execute .NET assembly in remote process.
+    parameters:
+      pid:
+        type: integer
+        description: The process ID to inject into.
+      assembly:
+        type: string
+        description: The assembly to inject.
+      args:
+        type: string
+        description: Arguments to pass to the assembly.
+    cmdline:
+      - assembly_inject
+      - -PID
+      - ${pid}
+      - -Assembly
+      - ${assembly}
+      - -Arguments
+      - ${args}
+  blockdlls:
+    description: Block non-Microsoft signed DLLs from loading into post-ex jobs.
+    parameters:
+      enable:
+        type: boolean
+        description: Enable or disable blockdlls.
+    cmdline:
+      - blockdlls
+      - -EnableBlock
+      - ${enable}
+    
+  cat: 
+    description: Retrieve the output of a file.
+    parameters:
+      file:
+        type: string
+        description: The file to retrieve.
+    cmdline:
+      - cat
+      - -Path
+      - ${file}
+  cd:
+    description: Change working directory.
+    parameters:
+      dir:
+        type: string
+        description: The directory to change to.
+    cmdline:
+      - cd
+      - -Path
+      - ${dir}
```

Comment:
same as above, might be worth creating a shell.yml with these commands, ideally relying on the [platforms conditional like here](https://github.com/dreadnode/robopages/blob/dffac3358961182ff4661effeec9f002348cbc50/cybersecurity/reverse-engineering/symbols.yml#L16)